io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+rename-icon: appimagepool
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: appimagepool
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - install -Dm644 appimagepool/appimagepool /app/appimagepool/appimagepool
+      - chmod +x /app/appimagepool/appimagepool
+      - cp -R appimagepool/usr/share /app/share
+      - cp -R appimagepool/lib /app/appimagepool/lib
+      - cp -R appimagepool/data /app/appimagepool/data
+      - sed -i 's/<id>appimagepool/<id>io.github.prateekmedia.appimagepool/' /app/share/metainfo/appimagepool.appdata.xml 
+      - mv /app/share/metainfo/appimagepool.appdata.xml /app/share/metainfo/io.github.prateekmedia.appimagepool.appdata.xml
+      - mkdir -p /app/bin/ /app/lib/
+      - ln -s /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0 /app/lib/libgio-2.0.so
+      - ln -s /app/appimagepool/appimagepool /app/bin/appimagepool
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/prateekmedia/appimagepool/releases/download/2.3.1/appimagepool-x86_64.tar.gz
```

Comment:
since you said it's ok on aarch64, why not building from source here? Flatpak do that well.

Reply:
The problem is I have to setup flutter here, do snap works inside flatpak container? I think the easiest way is to `snap install flutter` or I would have to or redownload flutter SDK and do all the manual steps.

Reply:
`snap` won't work.